### ISSUE TITLE
Lands are still defaulting to tapping for 1st ability

### ISSUE THREAD

**Issue by drmDev** [magefree/mage#1795]
Painlands (and likely other lands that can tap for various different mana) sometimes cannot be tapped for anything other than the 1st available mana. It seems the workaround at the moment is to disable the Mana Payment options, then flip them back on again.

**Comment by deanboyj** [comment]
This is also causing issue with the manlands.  Im sure westvale abbey probably doesnt work either but i havnt tried yet.

**Comment by guizonatto** [comment]
Mutavault and Raging Ravine not working.

**Comment by LevelX2** [comment]
It's most probably caused by the new option so:
To fix the land activate ability problem switch on and off again the new menu option on the battlefield context menu "**Mana payment/Use first mana ability when tapping lands**"

**Comment by LevelX2** [comment]
Westvale Abbey use of non mana abilities won't be fixed by this workaround.

**Comment by spjspj** [comment]
D'oh :(  Sorry for this.

**Comment by drmDev** [comment]
I know it might be a pain in the ass but I highly recommend we patch this
up and push a new client version ASAP. Lands not tapping for mana without
workarounds and even with them is pretty unplayable.

**Comment by spjspj** [comment]
From a comment in chat on xmage.de, turning it off and then on again seems to make it ok

**Comment by drmDev** [comment]
Also should check other client side changes while we're at it. The draft
changes and F11 hotkey. I'm out right now so can't test.

**Comment by spjspj** [comment]
F11 has been working fine I believe.  I use it fairly extensively in edh and it's worked every time and I know opponents I've played against use it

**Comment by spjspj** [comment]
Alt+1 can send out a lot of player action messages at the moment as well 
```
session.sendPlayerAction(PlayerAction.USE_FIRST_MANA_ABILITY_ON, gameId, null);
```

**Comment by drmDev** [comment]
The draft change works well too, I tested that recently. It can still be improved with timer adjustments - but that's a later improvement. As to this issue with lands, I think a backout of the feature is needed until a resolution is found. We are never going to hear the end of this since no one actually reads previously reported bugs. Hopefully we can push a client update with just that feature backed out.

**Comment by drmDev** [comment]
should be fixed by bfc66ff3f3202a906c77a0f0d79d5fd22ba69f6c and dacef8db5e8251b573bf6685a3f4917c46561df2
